### PR TITLE
zh-cn: init the translation of the `Document.createTouch()` method

### PR DESCRIPTION
**Pull request by cnbilinyj**
Chinese document for 'Document: creatTouch() method'

### Description

Chinese document for 'Document: creatTouch() method'

### Motivation

Translate the document 'Document: creatTouch() method' into Chinese for Chinese users to read.

将文档“Document: createTouch() method”翻译成中文以供中文使用者阅读。

### Additional details

No data or info.

### Related issues and pull requests

No data or info.

### CODE REVIEW

File: files/zh-cn/web/api/document/createtouch/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "Document: createTouch() 方法"
+short-title: createTouch()
+slug: Web/API/Document/createTouch
+page-type: web-api-instance-method
+status:
+  - deprecated
+  - non-standard
+browser-compat: api.Document.createTouch
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/document/createtouch/index.md:9]:
```suggestion
```

File: files/zh-cn/web/api/document/createtouch/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+---
+
+{{APIRef("DOM")}}{{Deprecated_Header}}{{Non-standard_header}}
+
+The **`Document.createTouch()`**方法创建并返回一个新的{{DOMxRef("Touch")}}对象。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/document/createtouch/index.md:14]:
```suggestion
**`Document.createTouch()`** 方法创建并返回一个新的 {{DOMxRef("Touch")}} 对象。
```

File: files/zh-cn/web/api/document/createtouch/index.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  - : {{DOMxRef("Touch.radiusY")}}的值。
+- `rotationAngle`
+  - : {{DOMxRef("Touch.rotationAngle")}}的值。
+- `force`
+  - : {{DOMxRef("Touch.force")}}的值。
+
+### 返回值
+
+- `touch`
+  - : 按照输入参数的描述配置的{{DOMxRef("Touch")}}对象。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/document/createtouch/index.md:66]:
```suggestion
- `view`
  - : 发生触摸的 {{DOMxRef("window")}}。
- `target`
  - : 触摸的 {{DOMxRef("EventTarget")}}。
- `identifier`
  - : {{DOMxRef("Touch.identifier")}} 的值。
- `pageX`
  - : {{DOMxRef("Touch.pageX")}} 的值。
- `pageY`
  - : {{DOMxRef("Touch.pageY")}} 的值。
- `screenX`
  - : {{DOMxRef("Touch.screenX")}} 的值。
- `screenY`
  - : {{DOMxRef("Touch.screenY")}} 的值。

> [!NOTE]
> 该方法的先前版本包括以下附加参数，但这些参数未包含在下面列出的任何标准中。因此，这些参数应被视为不推荐使用。

- `clientX`
  - : {{DOMxRef("Touch.clientX")}} 的值。
- `clientY`
  - : {{DOMxRef("Touch.clientY")}} 的值。
- `radiusX`
  - : {{DOMxRef("Touch.radiusX")}} 的值。
- `radiusY`
  - : {{DOMxRef("Touch.radiusY")}} 的值。
- `rotationAngle`
  - : {{DOMxRef("Touch.rotationAngle")}} 的值。
- `force`
  - : {{DOMxRef("Touch.force")}} 的值。

### 返回值

- `touch`
  - : 按照输入参数的描述配置的 {{DOMxRef("Touch")}} 对象。
```

File: files/zh-cn/web/api/document/createtouch/index.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## 例子
+
+此示例说明了如何使用`Document.createTouch()`方法创建{{DOMxRef("Touch")}}对象。
+
+在下面的代码片段中，为`target`元素创建了两个{{DOMxRef("Touch")}}对象。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/document/createtouch/index.md:72]:
```suggestion
此示例说明了如何使用 `Document.createTouch()` 方法创建 {{DOMxRef("Touch")}} 对象。

在下面的代码片段中，为 `target` 元素创建了两个 {{DOMxRef("Touch")}} 对象。
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]
请注意格式，参阅 https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

File: files/zh-cn/web/api/document/createtouch/index.md (hunk 4 above)
Comment (cnbilinyj) [reply files/zh-cn/web/api/document/createtouch/index.md:72]: 感谢更正！

**Review by yin1999 — APPROVED** [review]
Thanks @cnbilinyj. Welcome to the community :tada:

**Pull request merged**